tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -448,29 +449,6 @@ _handle_ingress_aks() {
 	popd
 }
 
-# Implements the ingress handler for servernode
```

Comment:
I think it's fine to keep the following comments lines.

Reply:
@wainersm changes applied